test/ranges.jl

```diff
@@ -710,13 +710,13 @@ function test_linspace_identity(r::Range{T}, mr) where T
     @test -collect(r) == collect(mr)
     @test isa(-r, typeof(r))
 
-    @test 1 + r + (-1) == r
-    @test 1 + collect(r) == collect(1 + r) == collect(r + 1)
-    @test isa(1 + r + (-1), typeof(r))
-    @test 1 - r - 1 == mr
-    @test 1 - collect(r) == collect(1 - r) == collect(1 + mr)
-    @test collect(r) - 1 == collect(r - 1) == -collect(mr + 1)
-    @test isa(1 - r - 1, typeof(r))
+    @test 1 .+ r .+ (-1) == r
+    @test 1 .+ collect(r) == collect(1 .+ r) == collect(r .+ 1)
+    @test_broken isa(1 .+ r .+ (-1), typeof(r))
```

Comment:
not really broken so much as not done by broadcast, since it's tough in general to know that an arbitrarily fused operation preserves spacing of a range